package.json

```diff
@@ -14,13 +14,19 @@
     "prettier-project": "yarn run prettier-write 'src/**/*.{ts,tsx,js,jsx}'",
     "prettier-write": "yarn run prettier --write",
     "mocha": "sh scripts/mocha.sh",
+    "jest": "sh scripts/jest.sh",
     "start": "sh scripts/start.sh",
     "production": "sh scripts/production.sh",
     "publish-assets": "sh scripts/publish-assets.sh",
     "task": "node -r coffeescript/register -r @babel/register -r dotenv/config",
     "test:watch": "yarn jest -- --watch --runInBand",
     "test": "sh scripts/test.sh",
-    "type-check": "tsc --pretty --noEmit"
+    "type-check": "tsc --pretty --noEmit",
+    "install-codecov": "curl -s https://codecov.io/bash > .codecov && chmod +x ./.codecov",
+    "normalize-jest-coverage": "node scripts/normalize-jest-coverage.js",
+    "create-merged-coverage": "nyc report --reporter=text-lcov > coverage.lcov",
+    "report-coverage": "./.codecov -C $(cat COMMIT_HASH.txt) -p app/ -f coverage.lcov",
```

Comment:
Do we want these files to be checked in, or is this something we'd want in the `.gitignore`?  Not sure if this is something we want available to all, or if it is more of a dev tool.

Reply:
which files? The COMMIT_HASH.txt have needed to be present for a bit to publish assets, but I could commit the codecov executable instead of `curl`-ing it down. I'll add coverage.lcov to the gitignore though incase someone runs this yarn script locally.

Reply:
I guess whichever files are generated by these commands -- looks like the `.lcov` files